main.js

```diff
@@ -15,5 +18,8 @@ function calculateWordsStatistic(text) {
   return collectStatistic(parseWords(text));
```

Comment:
Let's call `toString` here instead of insight `parseWords` function, so we will have:

```
return collectStatistic(parseWords(objectForAnalysis.toString()));
```

Reply:
Changes made